Source/Other/Extensions/DateFormatterExtension.swift

```diff
@@ -41,7 +41,7 @@ extension DateFormatter {
 	/// 2020-06-30 14:14:11 is  example
 	@objc public static var naturalLanguageFormatter: DateFormatter = {
 		let formatter = DateFormatter()
-		formatter.dateFormat = "yyyy-MM-dd hh:mm:ss"
+		formatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
```

Comment:
Shouldn't this probably be dependent on the user's localization settings? If it really is natural language, some parts of the world are 24hr time and others aren't

Reply:
nah, `hh` is 12 hours only, `HH` is both 12 and 24 hour ... that was my mistake, so it returned value only until noon 😂

Reply:
Right, but in America you would want 12 hour time suffixed by AM/PM lol. Although most devs should be comfy with 24hr time

Reply:
I don't think mysql can return that - this is for parsing the mysql format